mmv1/third_party/terraform/tests/resource_billing_budget_test.go

```diff
@@ -626,28 +611,14 @@ data "google_billing_account" "account" {
 	billing_account = "%{billing_acct}"
 }
 
-resource "google_project" "project1" {
-	project_id      = "tf-test-%{random_suffix_1}"
-	name            = "tf-test-%{random_suffix_1}"
-	org_id          = "%{org}"
-	billing_account = data.google_billing_account.account.id
-}
-
-resource "google_project" "project2" {
-	project_id      = "tf-test-%{random_suffix_2}"
-	name            = "tf-test-%{random_suffix_2}"
-	org_id          = "%{org}"
-	billing_account = data.google_billing_account.account.id
-}
-
 resource "google_billing_budget" "budget" {
 	billing_account = data.google_billing_account.account.id
 	display_name    = "Example Billing Budget"
 
 	budget_filter {
 		projects = [
-			"projects/${google_project.project2.number}",
-			"projects/${google_project.project1.number}",
+			"projects/%{project_2}",
+			"projects/%{project_1}",
```

Comment:
These projects need to be real. The test is failing because the api is cleansing them.